workspace-manager-clienttests/README.md

```diff
@@ -19,13 +19,13 @@ Note: `render-config.sh` assumes a valid `.vault-token` in your `$HOME` director
 To run the test, use the following command from the workspace-manager-clienttests directory.
 
 ```
-./gradlew  runTest --args="configs/BasicAuthenticated.json /tmp/TR"
+./gradlew  runTest --args="integration/BasicAuthenticated.json /tmp/TR"
```

Comment:
Aside: right now all `BasicAuthenticated` has is `GetWorkspace`, so I'm wondering if it needs a new name. It's a config, and not a suite, so maybe that's the confusion. Maybe change `Basic` to `Smoke`, since that's kinda what this test does, and it's referred to as a smoke test somewhere else.

Reply:
We can look at it this way. The `GetWorkspace` example has a simple user journey of retrieving a valid workspace. The user journey can be expanded to involve other APIs (e.g. Data Repo) but simply not in scope for this ticket.

For example, [WorkspaceIntegrationTest](https://github.com/DataBiosphere/terra-workspace-manager/blob/dev/src/test/java/bio/terra/workspace/integration/WorkspaceIntegrationTest.java) can be refactored using the test runner framework. The PF team might already been looking at it.